build/vsts-ci.yml

```diff
@@ -104,13 +104,61 @@ phases:
       artifactName: PackageAssets
       artifactType: container
 
+################################################################################
+- phase: Windows_x86
+################################################################################
+  variables:
+    BuildConfig: Release
+    OfficialBuildId: $(BUILD.BUILDNUMBER)
+    DOTNET_CLI_TELEMETRY_OPTOUT: 1
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+    DOTNET_MULTILEVEL_LOOKUP: 0
+    _SignType: real
+    _UseEsrpSigning: true
+    _TeamName: DotNetCore
+  queue:
+    name: DotNetCore-Build
+    demands: 
+      - agent.os -equals Windows_NT
+  steps:
+
+  - task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
+    displayName: Install MicroBuild Signing Plugin
+    inputs:
+      signType: '$(_SignType)'
+      zipSources: false
+      esrpSigning: '$(_UseEsrpSigning)'
+    env:
+      TeamName: $(_TeamName)
+    continueOnError: false
+    condition: and(succeeded(), in(variables._SignType, 'real', 'test'))
+
+  - script: ./build.cmd -$(BuildConfig) -buildArch=x86
```

Comment:
We should only be building the native assets on the Windows_x86 leg.  See the Linux and OSX legs.

`script: ./build.sh -buildNative -$(BuildConfig)`

The reason for this is because we don't want the IL assets to collide/conflict.  Each leg needs to build a unique set of assets.

Today:
OSX: builds the OSX native assets.
Linux: builds the Linux native assets
Windows: builds Windows x64 native assets, and the managed assets.

This new leg should only be building the Windows x86 native assets.

Reply:
Thank you for pointing that out! I changed the script so that it builds native assets only on this leg.